portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase

```diff
@@ -635,6 +635,84 @@ definition {
 		}
 	}
 
+	@description = "LPS-109991 Bug: When a variant is edited, other variants lose modifications"
+	@priority = "4"
+	test CheckContentPersistAfterEditVariant {
+		property portal.acceptance = "false";
+
+		task ("Create AB test and create first variant") {
+			ABTest.openABTestPanel();
+
+			ABTest.createABTest(
+				goal = "Bounce Rate",
+				testTitle = "AB test title");
+
+			ABTest.createVariant(variantName = "Variant Name 1");
+		}
+
+		task ("Edit variant page and add some fragment to the page") {
+			Click(
```

Comment:
this click should be in a macro, something like "ABTest.editVariant(ABTest.createVariant(variantName = "Variant Name 1")